pom.xml

```diff
@@ -236,6 +236,18 @@
         <version>${dependency.zeebe.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.camunda</groupId>
+        <artifactId>zeebe-scheduler</artifactId>
+        <version>${dependency.zeebe.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.camunda</groupId>
+        <artifactId>zeebe-stream-platform</artifactId>
+        <version>${dependency.zeebe.version}</version>
+      </dependency>
+
```

Comment:
:thought_balloon: I didn't add these, but I think since we'd disabled the warning we missed them being now required.